sdk/plugins/host/load.go

```diff
@@ -1,18 +1,14 @@
 package external_host_plugins
 
 import (
-	"bytes"
-	"compress/gzip"
 	"context"
 	"fmt"
 	"io/fs"
-	"io/ioutil"
-	"os"
-	"path/filepath"
-	"runtime"
 	"strings"
 
 	pb "github.com/hashicorp/boundary/sdk/pbs/plugin"
+	"github.com/hashicorp/go-plugin"
```

Comment:
Reviewers: The actual plugin instantiating code moved over to pluginutil where it's now shared between KMS and hosts (and can satisfy any other interface), hence all the red.